autoload/lsc/message.vim

```diff
@@ -2,6 +2,22 @@ function! lsc#message#show(message, ...) abort
   call s:Echo('echo', a:message, get(a:, 1, 'Log'))
 endfunction
 
+function! lsc#message#showRequest(message, ...) abort
```

Comment:
it's only called in one place with the same number of arguments each time so I don't think we need the `...`